src/main/drivers/max7456.c

```diff
@@ -910,11 +755,14 @@ void max7456HardwareReset(void)
     IOInit(max7456ResetPin, OWNER_OSD, 0);
     IOConfigGPIO(max7456ResetPin, IO_RESET_CFG);
 
-
-    // RESET
+    // RESET 50ms long pulse, followed by 100us pause
     IOLo(max7456ResetPin);
-    delay(100);
+    delay(50);
     IOHi(max7456ResetPin);
+    delayMicroseconds(100);
+#else
+    // Allow device 50ms to powerup
+    delay(50);
```

Comment:
Is this delay actually needed? Considering no currently supported targets seem to be having the MAX7456 reset pin connected (and only one target ever did), I am thinking of dropping this function alltogether.